ziti/enroll/enroll.go

```diff
@@ -60,6 +60,7 @@ type EnrollmentFlags struct {
 	AdditionalCAs string
 	Username      string
 	Password      string
+	Verbose       bool
```

Comment:
I don't see how this helps if it isn't referenced.

Reply:
it gets referenced in the `ziti` project. I need this, for the next PR